components/shared/LinkButton.js

```diff
@@ -12,6 +14,11 @@ const OutlineLink = styled.a`
   background-color: ${({ backgroundColor, theme }) =>
     backgroundColor ? theme.colors[backgroundColor] : theme.colors.white};
 
+  ${below.small`
+    &.stretch-sm {
```

Comment:
Thinking we should remove the class selector and just make this the default behavior.

Reply:
That should work.